src/tests/integration/omneo/rewards/delete-reward.test.ts

```diff
@@ -0,0 +1,91 @@
+import { describe, expect, test, afterAll } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import { RewardDefinitionCreateInput } from '../../../../types'
+import { getRandomString } from './util'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+const CREATED_REWARDS_DEFINITION_IDS : number[] = []
+const FAILED_REWARDS_IDS : number[] = []
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('Reward delete', () => {
+  test('SDK Reward delete', async () => {
```

Comment:
This test is testing specifically reward delete.
So the only thing that should be using the SDK is the deletion. 

Everything else should use simpleOmneoRequest